BingMaps/sdk-native/getting-started-android/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Getting Started with Android | Microsoft Docs"
-author: "pablocan"

```

Comment:
I am not sure if we want both, author and ms.author. If you look at other files, (e.g. BingMaps/index.md), it contains both.  I remember I added title/author to fix warnings. Also, we may want to add description as they may change from suggestion to warnings in the future (although this can be in separate check in).

Reply:
Hmm, I don't see "author" in BingMaps/index.md

Reply:
In https://raw.githubusercontent.com/MicrosoftDocs/bingmaps-docs/master/BingMaps/index.md?